systemvm/debian/opt/cloud/bin/setup/router.sh

```diff
@@ -18,35 +18,46 @@
 
 . /opt/cloud/bin/setup/common.sh
 
-setup_router() {
-  log_it "Setting up virtual router system vm"
+check_reboot() {
+  if [ "$HYPERVISOR" != "vmware" ]; then
```

Comment:
We can skip this check and reboot whenever mngmt net is not available.

Reply:
copy will fix, I added this as it was very specific to vmware.

Reply:
I checked this check is very specific to vmware which uses the private/mgmt nic for programming VR over port 3922, therefore we don't need to do it for other hypervisors.